app/javascript/spotlight/admin/blocks/resources_block.js

```diff
@@ -8,18 +8,16 @@ Core.Block.Resources = (function(){
     formable: true,
     autocompleteable: true,
     show_heading: true,
-    show_alt_text: true,
```

Comment:
I tried to mirror description and not make this a separate function. Every time I did I would get an error.